cip-00XX/cip-00XX.md

```diff
@@ -0,0 +1,58 @@
+## CIP 00XX
+
+<pre>
```

Comment:
@hythloda isn't this a duplicate of https://github.com/global-synchronizer-foundation/cips/pull/55/files but with slightly different content?

Reply:
Thanks! closing!